Tickers/Tickers/UIComponents/CardPetView/CardPetView.swift

```diff
@@ -8,13 +8,85 @@
 import SwiftUI
 
 struct CardPetView: View {
+    
+    var isLocked:Bool
+    
+    var actualProgress: Int
+    var totalProgress: Int
+    @State var tickerName: String
+    var tickerLevel: Int
+    var tickerImage: String
+    var tickerEgg: String
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack {
+            RoundedRectangle(cornerRadius: 20)
+                .fill(Color("White"))
+                .frame(width: 160, height: 176)
+                .overlay {
+                    cardPet(isLocked:true)
+                }
+        }
     }
+    
+    @ViewBuilder func cardPet(isLocked: Bool) -> some View {
+        if isLocked {
+            ZStack {
+                Image(tickerEgg)
+                Image("padlock")
+                    .offset(x: 55, y: 60)
+            }
+        } else {
+            VStack {
+                HStack {
+                    Text(tickerName)
+                        .font(.custom("Nunito-Bold", size: 12))
+                        .foregroundColor(Color("Blue2"))
+                    Image("pencilWhite")
+                        .padding([.leading], -4)
+                }
+                .padding([.top], 10)
+                .padding([.leading], 4.5)
+
+                Spacer()
+                    .frame(height: 0)
+
+                Image(tickerImage)
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 105, height: 100)
+                    .padding([.leading], 17)
+
+                Spacer()
+                    .frame(height: 1)
+
+                HStack{
+                    Text("Lvl.")
+                        .font(.custom("Nunito-Bold", size: 12))
```

Comment:
já é possivel usar a `tickerFont` no projeto em vez de usar uma customizavel 
ex: `.tickerFont(size: 12, weight: .regular)` // ou bold se for bold